addons/controls/README.md

```diff
@@ -0,0 +1,419 @@
+<center>
+  <img src="./docs/media/addon-controls-hero.gif" width="100%" />
+</center>
+
+<h1>Storybook Controls</h1>
+
+Storybook Controls is UI to interactively edit your component stories. Stories written for controls are:
```

Comment:
```suggestion
Storybook Controls enables you to interact with a component's inputs dynamically, without needing to code. It creates an addon panel for you to edit inputs live.

Stories written for controls are:
```

Reply:
Address these objections: 
- "Do I have to change the way I write components for Controls?"
- "Do controls and args lock me into an API?"

The answers can be yes, but it helps to know straightaway.